cpp/src/arrow/compute/kernels/scalar_round_benchmark.cc

```diff
@@ -45,6 +45,7 @@ static void RoundArrayBenchmark(benchmark::State& state, const std::string& func
       rand.Numeric<ArrowType>(array_size, min, max, args.null_proportion));
   RoundOptions options;
   options.round_mode = static_cast<RoundMode>(Mode);
+  options.ndigits = 1;  // Round to the appropriate tenth.
```

Comment:
Looks the default `ndigits=0` is the common case.

Reply:
For floating point types, yes.   For integer types, `ndigits=0` should be optimized away.  I suppose I could set it up to run 0/1 for floating point and just 1 for integer types.  Not that it makes a difference here but the two arg version of round (the version that will be exposed to Substrait) is going to be using random values from -6 to +6.